### PR TITLE
4157 Give HTML classes to stats items

### PR DESCRIPTION
**Pull request by sarken**
https://code.google.com/p/otwarchive/issues/detail?id=4157

Give the various types of stats (kudos, comments, hits, etc) a dedicated class name so users can `display: none;` them instead of us needing to support preferences for such things.

### CODE REVIEW

File: app/views/stats/index.html.erb [hunk 1]
```diff
@@ -24,12 +24,20 @@
 <div class="wrapper">
   <dl class="statistics meta group" role="complementary">
     <% [:user_subscriptions, :kudos, :comments, :bookmarks, :subscriptions, :word_count].each do |stat| %>
-      <dt><%= (stat.to_s.titleize == "Comments") ? ts("Comment Threads") : stat.to_s.titleize %></dt>
-      <dd><%= @totals[stat] %></dd>
+      <% if stat.to_s.titleize == "User Subscriptions"
+          html_class = 'user subscriptions'
```
Comment (CristinaRO) [inline app/views/stats/index.html.erb]: If it's intentional that `html_class` in this case becomes two classes, the conditional could be simplified to have only one exception branch (the Word Count case), and the rest to assign `html_class = stat.to_s.humanize.downcase` (`humanize` gets rid of any underscores, but upcases the first letter, so `downcase` to lowercase everything).

And I know this is pre-existent code, but I can't think of a reason why the whole comparison couldn't be done on symbols to begin with. By which I mean:
```
<% if stat == :word_count
    html_class = 'words'
  else
    html_class = stat.to_s.humanize.downcase
  end
%>
<dt class="<%= html_class %>"><%= (stat == :comments) ? ts("Comment Threads") : stat.to_s.titleize %></dt>
```

**Comment by sarken** [comment]
Updated to be more better!

**Comment by CristinaRO** [comment]
I should know this, but I'm going to ~delegate: is this change going to need modifying the cache key for the blurb in order to take effect for existing blurbs?

**Comment by sarken** [comment]
Oops, yes -- some of the stats are in the cached area. I shall fix that.

**Comment by sarken** [comment]
Expired the cache on the blurb, but the stats are outside the cached area on the meta, so I didn't do anything there.

**Comment by sarken** [comment]
Closing because pull request #2010 contains the same code and has already been reviewed.

**Pull request closed without merging**